src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala

```diff
@@ -86,6 +86,19 @@ class FileRouteSpec extends AnyFlatSpec with Matchers with TestRequest with Befo
     response.errors.head.extensions.get.code should equal(expectedResponse.errors.head.extensions.get.code)
   }
 
+  "The api" should "throw an error if the consignment already has had files uploaded" in {
+    val sqlConsignment = s"insert into Consignment (SeriesId, UserId) VALUES (1,'$userId')"
+    val psConsignment: PreparedStatement = DbConnection.db.source.createConnection().prepareStatement(sqlConsignment)
+    psConsignment.executeUpdate()
+    //Seed DB with initial file for consignment
+    runTestMutation("mutation_one_file", validUserToken())
+
+    val expectedResponse: GraphqlMutationData = expectedMutationResponse("data_error_previous_upload")
+    val response: GraphqlMutationData = runTestMutation("mutation_one_file", validUserToken())
+    response.errors.head.message should equal (expectedResponse.errors.head.message)
```

Comment:
Once you've added a new GraphQL error code, it would be good to check that here too.

Reply:
Added test for the new error code